LICENSING.md

```diff
@@ -15,7 +15,6 @@ packages/grafana-e2e-selectors/
 packages/grafana-runtime/
 packages/grafana-toolkit/
 packages/grafana-ui/
-packages/jaeger-ui-components/
```

Comment:
I am not sure that at this moment we can just say it's no longer apache license as we mostly just moved it. I mean this would change the license to AGPL-3 and not sure if Apache license permits that (mind that as it's not all our code we cannot relicense it if license does not permit it I think)

Reply:
i was wondering the same, according to https://www.apache.org/foundation/license-faq.html#Distribute-changes

>I'VE MADE IMPROVEMENTS TO THE APACHE CODE; MAY I DISTRIBUTE THE MODIFIED RESULT?
Absolutely -- subject to the [terms of the Apache license](https://www.apache.org/licenses/LICENSE-2.0#redistribution), of course. You can give your modified code away for free, sell it, keep it to yourself, or whatever you like. Just remember that the original code is still covered by the Apache license and you must comply with its terms. Even if you change every single line of the Apache code you're using, the result is still based on the Foundation's licensed code. You may distribute the result under a different license, but you need to acknowledge the use of the Foundation's software. To do otherwise would be stealing.
If you think others would find your changes useful, though, we do encourage you to submit them to the appropriate Apache project for possible inclusion.

and the [terms](https://www.apache.org/licenses/LICENSE-2.0#redistribution) makes it look quite clear to me that we can't just relicense it 🤔